### PR TITLE
Card file variant

### PR DESCRIPTION
**Pull request by WengerK**

### CODE REVIEW

File: assets/sass/molecules/_card.scss [hunk 1]
```diff
@@ -231,6 +233,7 @@ a.card-img-top {
 
 .card-white { @include card-bg-variant($white); }
 .card-berrylight-gray { @include card-bg-variant($berrylight-gray); }
+.card-no-bg { background-color: transparent; }
```
Comment (tonifisler) [inline assets/sass/molecules/_card.scss]: It's never used. We can remove it.

File: assets/sass/molecules/_card.scss [hunk 2]
```diff
@@ -231,6 +233,7 @@ a.card-img-top {
 
 .card-white { @include card-bg-variant($white); }
```
Comment (tonifisler) [inline assets/sass/molecules/_card.scss]: Just noticed this class is useless. To be removed from V3 and from all Drupal instances.

File: assets/components/molecules/card-file.html.swig [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+title: Card File
+name: card-file
+collection: molecules
+variants:
+  grp: Group files
+notes: |
+  This is an implementation of the card component.
+---
+<div class="card card-no-border card-no-padding card-no-shadow">
+  <div class="card-body">
```
Comment (tonifisler) [inline assets/components/molecules/card-file.html.swig]: Why do we need to put it in a card component? It's using basically nothing from it 🤔 

I tested by removing the `.card` and `.card-body` wrappers and nothing changes. Delete them!

**Comment by WengerK** [comment]
@tonifisler last review :D ?

**Pull request merged**